packages/enhanced/src/lib/container/runtime/FederationRuntimePlugin.ts

```diff
@@ -193,6 +201,10 @@ class FederationRuntimePlugin {
         ),
       )}`;
     }
+
+    if (!existedFilePath) {
+      onceForCompilerEntryMap.set(compiler, this.entryFilePath);
```

Comment:
Is there going to be a way to mark a entryFilePath as changed?

Reply:
entry file path should never change since its a virtual module we generate, any change to it would require the compiler to restart anywys since the entry file is generated during compiler startup